apps/SageAccountWeb/src/App.tsx

```diff
@@ -127,15 +127,17 @@ function App() {
           element={<ResetPassword returnToUrl="/authenticated/myaccount" />}
         />
         <Route path={RESET_2FA_ROUTE} element={<ResetTwoFactorAuth />} />
-        <Route
-          path="authenticated/*"
-          element={
-            <>
-              <AuthenticatedRoutes />
-              <Footer />
-            </>
-          }
-        />
+        <Route path={'authenticated'}>
+          <Route
+            path="*"
+            element={
+              <>
+                <AuthenticatedRoutes />
+                <Footer />
+              </>
+            }
+          />
+        </Route>
```

Comment:
Had to nest `Route`s here in one of the v7 breaking changes. I think it only affects behavior if, within this route tree, we were to use a relative path in a Link component, which we do not do.